pkg/common/cns-lib/volume/manager.go

```diff
@@ -795,7 +843,7 @@ func (m *defaultManager) CreateVolume(ctx context.Context, spec *cnstypes.CnsVol
 		}
 		// Call CreateVolume implementation based on FSS value.
 		if m.idempotencyHandlingEnabled {
-			return m.createVolumeWithImprovedIdempotency(ctx, spec)
+			return m.createVolumeWithImprovedIdempotency(ctx, spec, extraParams)
```

Comment:
Is enhanced idempotency not supported by default?

Reply:
We have enabled it by default for vanilla but the FSS still exists in the configmap for WCP.